M17_spec.tex

```diff
@@ -849,18 +849,18 @@ \section{Packet Mode}
 		\hline
 		Bytes & Meaning \\
 		\hline
-		1..798 & Application Packet Data \\
-		2 & CRC \\
+		0..822 & Application Packet Data \\
+		823..824 & CRC \\
```

Comment:
These are quantities, not byte numbers, right?
```suggestion
		1..823 & Application Packet Data \\
		2 & CRC \\
```
I.e. the CRC comes right after the last packet byte?

Reply:
I suppose it could also be written as byte numbers: `0..n-1`, `n..n+1`, where `n` is the packet length.

Reply:
These are locations.